resources/aws_ec2.go

```diff
@@ -483,36 +464,15 @@ func (obj *AwsEc2Res) longpollWatch() error {
 						event: event,
 					}:
 					case <-obj.closeChan:
+						return
```

Comment:
We don't want to return in case of an event, just on closeChan. I think I was looking at the wrong select when I added this. Either way, I should have thought about it more. :\